src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,71 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engine;
+    private String name;
+    private String color;
+    private int issueYear;
+    private int coast;
 
+    public Engine getEngine() {
+        return engine;
+    }
+
+    public void setEngine(Engine engine) {
+        this.engine = engine;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getIssueYear() {
+        return issueYear;
+    }
+
+    public void setIssueYear(int issueYear) {
+        this.issueYear = issueYear;
+    }
+
+    public int getCoast() {
+        return coast;
+    }
+
+    public void setCoast(int coast) {
+        this.coast = coast;
+    }
+
+    @Override
+    public String toString() {
+        return "Car{"
+                + "engine=" + engine
+                + ", name='" + name + '\''
+                + ", color='" + color + '\''
+                + ", issueYear=" + issueYear
+                + ", coast=" + coast
+                + '}';
+    }
+
+    @Override
+    public Car clone() {
```

Comment:
Do you have a deep copy or a shallow copy?

Reply:
forgot to call method clone(), fixed